src/test/java/selenium/webdriver/AlertsTest.java

```diff
@@ -0,0 +1,63 @@
+package selenium.webdriver;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class AlertsTest {
+    private WebDriver driver;
+    @BeforeEach
+    public void setup() {
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--ignore-ssl-errors=yes", "--ignore-certificate-errors");
+        driver = new ChromeDriver(options);
+    }
+
+    @Test
+    @DisplayName("JavaScript Confirm Box Test")
+    public void alertBox() throws InterruptedException {
```

Comment:
method name must contain word test in the beginning or in the end of the method name

Reply:
Done